src/main/java/seedu/duke/TeamMemberList.java

```diff
@@ -0,0 +1,38 @@
+package seedu.duke;
+
+import java.util.ArrayList;
+
+public class TeamMemberList {
+    protected static ArrayList<TeamMember> memberList = new ArrayList<>();
+
+    public TeamMemberList() {
+
+    }
```

Comment:
Should the creating of a new instance be inside this function instead?
public TeamMemberList() {
     members = new ArrayList<>();
}

Reply:
dont need cause the variable is already initialised